cmd/kusk/cmd/generate.go

```diff
@@ -299,3 +239,56 @@ func init() {
 
 	apiTemplate = template.Must(template.New("api").Parse(templates.APITemplate))
 }
+
+var generateDescription = `Description:
+
+Generate accepts your OpenAPI spec file as input either as a local file or a URL and generates a Kusk 
+Gateway compatible API resource that you can apply directly into your cluster. 
+
+It does this via the x-kusk extension which will be added automatically if one is not already set. It will 
+set the upstream service, namespace and port to the flag values passed, respectively, and set the rest of 
+the settings to defaults.
+
+If the x-kusk extension is already present, it will override the upstream service, namespace and port to 
+the flag values passed, respectively, and leave the rest of the settings as they are.`
+
+var generateHelp = `
+No name specified:
+kusk api generate \
+	-i spec.yaml \
+	--envoyfleet.name kusk-gateway-envoy-fleet \
+	--envoyfleet.namespace kusk-system
+
+In the above example, kusk will use the openapi spec info.title to generate a manifest name and leave the existing
+x-kusk extension settings
+
+No API namespace specified:
+kusk api generate \
+	-i spec.yaml \
+	--name httpbin-api \
+	--upstream.service httpbin \
+	--upstream.port 8080 \
+	--envoyfleet.name kusk-gateway-envoy-fleet
+
+In the above example, as --namespace isn't defined, it will assume the default namespace.
+
+Namespace specified:
+kusk api generate \
+	-i spec.yaml \
+	--name httpbin-api \
+	--upstream.service httpbin \
+	--upstream.namespace my-namespace \
+	--upstream.port 8080 \
+	--envoyfleet.name kusk-gateway-envoy-fleet
+
+OpenAPI definition form URL:
+This will fetch the OpenAPI document from the provided URL and generate a Kusk Gateway API resource
+
+kusk api generate \
```

Comment:
Simplifying a little bit and use a valid example: 

```sh
kusk generate \
	-i https://raw.githubusercontent.com/kubeshop/kusk-gateway/main/examples/todomvc/spec/todospec.yaml
```

Reply:
All examples and text I used were already there. I didn't look into the content just formatting.

So please add/remove whatever you think is necessary.